src/components/slidedeck/index.js

```diff
@@ -3,36 +3,9 @@ import Slide from "./_slide";
 import style from "./index.scss";
 
 export default class slideDeck extends Component {
-  constructor(){
-    super();
-    this.state = {};
-
-    return this;
-  }
   onComplete(){
     this.props.fetch();
   }
-  screen(){
```

Comment:
Haha sorry, was looking to borrow some code from it and noticed it wasn't getting used!